### ISSUE TITLE
[Loki-Distrubted]  update helm chart for loki app version 3.X.X

### ISSUE THREAD

**Issue by sunilkumar-nfer** [grafana/helm-charts#3242]
Hi team,

When can we expect 3.x.x version of loki distributed helm chart. Do we have an estimated ETA for the official release?

**Comment by p-pal** [comment]
Agreed, with that we would be able to implement explore logs with distributed version.

**Comment by tobernguyen** [comment]
I think we will need to migrate to the [Loki helm chart](https://github.com/grafana/loki/tree/main/production/helm/loki) for v3 upgrade. In their [document](https://grafana.com/docs/loki/latest/setup/upgrade/upgrade-to-6x/), they said

> Separate instructions on how to enable this as well as how to migrate from the existing community chart will be coming shortly.

I'm not sure when they will release the migration guide.

**Comment by sunilkumar-nfer** [comment]
Hi Team,

Do we have any update on this?

**Comment by ksauzz** [comment]
It seems the migration guide is now available. If `loki-distributed` actually will be deprecated, I hope Grafana team would mark this chart as deprecated, and announce it.

https://grafana.com/docs/loki/latest/setup/migrate/migrate-from-distributed/
